src/services/goalService.ts

```diff
@@ -1,5 +1,6 @@
-import { Request, Response } from "express"
```

Comment:
Remove this file and make the changes per crud operation in separate branches (need to review each operation separately)